README.md

```diff
@@ -98,15 +98,123 @@ This release extends support for RHEL v8.5.
 
 ### Supported GPUs
 
-#### Radeon Pro V620 and W6800 Workstation GPUs
+Support can be assumed for current server accelerators GPUs intended for enterprise customers.
+This ROCm release officially supports workstation GPU products.
+However, there are also compatible consumer GPU products available.
+Finally, this ROCm release can be used with a number of older GPU products.
 
-This release extends ROCm support for Radeon Pro V620 and W6800 Workstation GPUs.
+#### Enterprise GPUs
 
-- SRIOV virtualization support for Radeon Pro V620
-- KVM Hypervisor (1VF support only) on Ubuntu Host OS with Ubuntu, CentOs, and RHEL Guest
-- Support for ROCm-SMI in an SRIOV environment. For more details, refer to the ROCm SMI API documentation.
+Current enterprise GPU accelerators can be assumed to be supported. These include:
+
+- AMD Instinct™ MI100
+- AMD Instinct™ MI200
+- AMD Instinct™ MI250
+- AMD Instinct™ MI250X
+
+#### Supported Workstation GPUs
+
+This ROCm release extends support to two workstation GPU products:
 
-**Note:** Radeon Pro v620 is not supported on SLES.
+- AMD Radeon™ PRO V620
+- AMD Radeon™ PRO W6800
+
+These features are verified to work by quality assurance (QA):
+
+- SRIOV virtualization support for AMD Radeon™ PRO V620
+- KVM Hypervisor (1VF support only) on Ubuntu Host OS with Ubuntu, CentOs, and RHEL Guest
+- Support for ROCm-SMI in an SRIOV environment. For more details, refer to [the ROCm SMI API documentation](https://rocmdocs.amd.com/en/latest/ROCm_System_Managment/ROCm-System-Managment.html).
+
+**Note:** AMD Radeon™ PRO V620 is not supported on SLES.
+
+#### Compatible Consumer GPUs
+
+The supported workstation GPU products all use the `gfx1030` instruction set architecture (ISA) implemented in the "Navi 21" silicon.
+The same silicon type implementing the same ISA is also used in some consumer GPU products, which *should* therefore be compatible with this ROCm release.
+However, ROCm was not validated against these products before release.
```

Comment:
It is unclear how this sentence is connected to the following. Do you mean something like:
```suggestion
However, ROCm was not validated against the following products before release:
```
or the following list is just an example of devices?

Reply:
No, you are right. The following table should be a complete matrix of devices and their library targets. I will commit your changes.